### PR TITLE
Update to sbt-paradox-akka

### PR DESCRIPTION
**Pull request by 2m**
Paradox release from
https://github.com/lightbend/paradox/commit/45cf0c39590811c536e94b6725307dfac2198673

Akka theme released from
https://github.com/akka/akka-paradox/commit/8f75a008910c50e2c0dcdd2f1704cceb23e917d0

### CODE REVIEW

File: akka-docs/build.sbt [hunk 1]
```diff
@@ -22,8 +22,4 @@ paradoxProperties ++= Map(
   "snip.akka.base_dir" -> ((baseDirectory in Test).value / "..").getAbsolutePath
 )
 
-resolvers += Resolver.bintrayRepo("2m", "maven")
-paradoxTheme := Some("com.lightbend.akka" % "paradox-theme-akka" % "b74885b8+20170511-1711")
-paradoxNavigationDepth := 1
-paradoxNavigationExpandDepth := Some(1)
-paradoxNavigationIncludeHeaders := true
```
Comment (raboof) [inline akka-docs/build.sbt]: Are these settings no longer required?

Comment (2m) [inline akka-docs/build.sbt]: They are part of sbt-paradox-akka now: https://github.com/akka/akka-paradox/commit/33fe4a724e4ff98e6440395bc6797b6ff28d935a#diff-cd79816a71bd31caf7a78b4ec42412ceR20

File: akka-docs/build.sbt [hunk 2]
```diff
@@ -22,8 +22,4 @@ paradoxProperties ++= Map(
   "snip.akka.base_dir" -> ((baseDirectory in Test).value / "..").getAbsolutePath
 )
 
-resolvers += Resolver.bintrayRepo("2m", "maven")
```
Comment (raboof) [inline akka-docs/build.sbt]: isn't this one still needed to fetch the theme?

Comment (2m) [inline akka-docs/build.sbt]: With the default config in akka-paradox the theme was published to sbt-plugin repo. So I did not reconfigure it and kept it as it was.

**Comment by 2m** [comment]
Merged as a part of https://github.com/akka/akka/pull/22919

**Pull request closed without merging**